### PR TITLE
Update example config for Spark

### PR DESCRIPTION
**Pull request by alonl**
* Fixed regex escaping
* Add Structured Streaming metrics

@brian-brazil

### CODE REVIEW

**Comment by alonl** [comment]
1. The current example didn't even work - so I escaped the dots in the metrics' names and now it's working.
2. Added scraping for Spark Structured Streaming metrics (See: https://issues.apache.org/jira/browse/SPARK-17731)
3. Regarding `lowercaseOutputName` and `attrNameSnakeCase`, I was actually not 100% sure about that.

File: example_configs/spark.yml [hunk 1]
```diff
@@ -1,40 +1,53 @@
+---
+lowercaseOutputName: true
```
Comment (brian-brazil) [inline example_configs/spark.yml]: I wouldn't add these, I'm starting to feel that munging names causes more confusion than it does clarity

Comment (alonl) [reply example_configs/spark.yml]: Delete both `lowercaseOutputName` and `attrNameSnakeCase` or just the former?

Comment (brian-brazil) [reply example_configs/spark.yml]: Both please, they weren't here before.

**Comment by alonl** [comment]
@brian-brazil I've updated the PR.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**